src/bundle/Resources/public/scss/_icons.scss

```diff
@@ -7,11 +7,26 @@
         height: calculateRem(16px);
     }
 
+    &--small-medium {
```

Comment:
I think that `medium-small` would be more pronounceable.

Reply:
Me and @dew326 agreed to use this name